amulet/sentry.py

```diff
@@ -120,10 +120,11 @@ def upload_scripts(self):
 
         # copy one at a time b/c `juju scp -r` doesn't work (currently)
         for f in glob.glob(os.path.join(source, '*')):
-            cmd = "juju scp {} {}:{}".format(
-                os.path.join(source, f),
-                self.info['unit_name'], dest)
-            subprocess.check_call(cmd.split())
+            src = os.path.join(source, f)
+            if os.path.isfile(src):
```

Comment:
Ah, okay. Maybe a comment to that effect? Otherwise, if you're just looking at the source tree and this code, it's not clear why this condition is needed.